src/views/blocks/attendees/view-link.php

```diff
@@ -61,13 +64,13 @@
 	return false;
 }
 
-$link = $view->get_tickets_page_url( $event_id, $is_event_page );
+$link = $view->get_tickets_page_url( $post_id, $is_event_page );
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $link

<sub>hash: bf2e95aff39d39199182b17dd845238e</sub>